wrappers/javascript/indy-vdr-react-native/native/indy_vdr.node

```diff
@@ -1,3 +0,0 @@
-# This is a placeholder file to prevent node-pre-gyp from installing the indy_vdr binary when cloning
```

Comment:
This needs to stay!

Reply:
sorry about that, I was testing some things in nodejs and didn't mean to commit it